### ISSUE TITLE
Linkedin post error: 403 Forbidden

### ISSUE THREAD

**Issue by brycekirk** [engram-design/SocialPoster#25]
### Description
Linked in produces the following response when I attempt to post:
```
{
    "statusCode": 403,
    "reasonPhrase": "Forbidden"
}
{
    "serviceErrorCode": 100,
    "message": "Not enough permissions to access: GET \/me",
    "status": 403
}
```
- Callback URL is correct
- App Permissions are r_emailaddress, r_liteprofile and w_member_social
- Company page is verified

### Steps to reproduce

1. Make an app and set up the Provider in Social Poster
2. Connect a Linkedin account
3. Go to an entry, enable the account and save
4. 403 error appears

### Additional info

- Plugin version: 2.0.8
- Craft version: Craft Pro 3.1.34

**Comment by zoinia** [comment]
Same issue.

**Comment by engram-design** [comment]
Looks like Linked.in's behaviour has changed. Investigating

**Comment by engram-design** [comment]
Fixed in [2.0.10](https://github.com/verbb/social-poster/releases/tag/2.0.10). Please note that you'll probably need to disconnect and connect the Linked.in provider again due to changed permissions.
